action.yml

```diff
@@ -0,0 +1,9 @@
+name: "Dependabot AutoMerge"
+description: "Allows to automatically review and merge Dependabot's pull requests."
+inputs:
+  github-token:
+    description: "A Githib Personal Access Token"
+    required: true
+runs:
+  using: 'node16'
```

Comment:
maybe try 18 when you get the chance :p

Reply:
I already tried and it doesn't work.
It asks to downgrade to `<= node16`